src/config.js

```diff
@@ -313,7 +313,8 @@ function parseArgs(options, raw_args) {
     });
 
     const runner_group = parser.addArgumentGroup({title: 'Test runner'});
-    const concurrency_default = '4+cpus';
+    // Default concurrency is number of cpu cores -1 for the main thread
+    const concurrency_default = getCPUCount() - 1;
```

Comment:
`-1` doesn't look right; what if I run this on a Raspberry Pi? Then it would be 0? How about just `'cpus'`? It's a string here intentionally, since that makes it show up in the help output, which may be generated and viewed on different machines.

Reply:
Good point, updated the PR 👍